Zilon.Core/CDT.LAST.MonoGameClient/StartUp.cs

```diff
@@ -43,13 +43,7 @@ protected override void RegisterSchemeService(IServiceCollection container)
         {
             container.AddSingleton((Func<IServiceProvider, ISchemeLocator>)(factory =>
             {
-                var mainModule = Process.GetCurrentProcess().MainModule;
-                if (mainModule is null)
-                {
-                    throw new InvalidOperationException("Error during main module calculation.");
-                }
-
-                var gamePath = mainModule.FileName;
+                var gamePath = Assembly.GetExecutingAssembly().Location;
```

Comment:
See fix in #1129
You should revert your changes and merge changes from pointed PR/branch.

Reply:
I will remove it by myself.